### DISCUSSION TITLE
command failed to run

### DISCUSSION THREAD

**Discussion by 0day404** [ricklamers/gpt-code-ui#35] [Q&A]
<img width="863" alt="屏幕截图 2023-06-07 051828" src="https://github.com/ricklamers/gpt-code-ui/assets/128878325/bbe703f7-8dac-4fe1-8827-6862544c745e">
<img width="872" alt="image" src="https://github.com/ricklamers/gpt-code-ui/assets/128878325/fea27b79-1643-49c7-a6a7-2b5f0da24fad">

**Comment by ricklamers** [comment]
Please run `pip install gpt-code-ui` first

**Reply by 0day404** [reply]
I have run it, only that command can be run

**Reply by 0day404** [reply]
> `pip install gpt-code-ui`请先跑

<img width="869" alt="屏幕截图 2023-06-07 110822" src="https://github.com/ricklamers/gpt-code-ui/assets/128878325/6da694c0-b4b2-468d-9a2d-222a1a3d3054">

**Reply by ricklamers** [reply]
https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-to-the-user-site

Likely your PATH environment variable does not include the directory to which `pip` installs packages.

To use `make` and `.env.example` cloning the repository is necessary.

**Comment by 0day404** [comment]
I'm still a freshman and it's hard to understand what you mean, please help me more, please
        
    
        
            
        
    
         
        
            
                        ***@***.***
            
        
     
        
  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            Rick ***@***.***>
            
        
      
      
        
         Date 
        
        
        6/7/2023 11:47
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            chen ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [ricklamers/gpt-code-ui] command failed to run (Discussion #35)
        
      
    
  

https://packaging.python.org/en/latest/tutorials/installing-packages/#installing-to-the-user-site
Likely your PATH environment variable does not include the directory to which pip installs packages.
To use make and .env.example cloning the repository is necessary.

**Reply by ricklamers** [reply]
This information should be easily found on Stack Overflow, Google or even ask ChatGPT :)

**Comment by 0day404** [comment]
What I mean is that even if I do that, it still can’t run. I don’t know where the problem is. Since I uninstalled python on the c drive and reinstalled it to the d drive, it has never been successful, whether it is chatgpt or New bing didn't solve my problem,
        
            
        
    
         
        
            
                        ***@***.***
            
        
     
        
  
    ---- Replied Message ----
  
  
    
      
        
         From 
        
        
            Rick ***@***.***>
            
        
      
      
        
         Date 
        
        
        6/7/2023 21:51
        
      
      
        
         To 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         Cc 
        
        
          
            chen ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         Subject 
        
        
              Re: [ricklamers/gpt-code-ui] command failed to run (Discussion #35)
        
      
    
  

This information should be easily found on Stack Overflow, Google or even ask ChatGPT :)
